README.md

```diff
@@ -1,344 +1,57 @@
 # Analytics-Swift
 
-NOTE: This project is currently in the Pilot phase and is covered by Segment's [First Access & Beta Preview Terms](https://segment.com/legal/first-access-beta-preview/).  We encourage you
-to try out this new library. Please provide feedback via Github issues/PRs, and feel free to submit pull requests.  This library will eventually 
-supplant our `analytics-ios` library, but customers should not use this library for production applications during our Pilot phase. 
+NOTE: This project is currently in the Beta phase and is covered by Segment's [First Access & Beta Preview Terms](https://segment.com/legal/first-access-beta-preview/).  We encourage you
+to try out this new library. Please provide feedback via Github issues/PRs, and feel free to submit pull requests.
 
-The hassle-free way to add Segment analytics to your Swift app (iOS/tvOS/watchOS/macOS/Linux).
+The hassle-free way to add Segment analytics to your Swift app (iOS/tvOS/watchOS/macOS/Linux). Analytics helps you measure your users, product, and business. It unlocks insights into your app's funnel, core business metrics, and whether you have product-market fit.
```

Comment:
If we're going to list all of the platforms, we should also include iPadOS.

Reply:
👍